lib/support/stop.js

```diff
@@ -8,5 +8,5 @@ module.exports = function(processName) {
 
   log.debug('Kill all processes ' + processName);
 
-  return execa('pkill', scriptArgs).catch();
+  return execa('pkill', scriptArgs).catch(() => {});
```

Comment:
This is to not handle the case when pkill exits with != 0, right? Which I guess is an acceptable error we don't want to cause browsertime to stop for.
If so I think using the reject=false option for execa is better, see https://github.com/sindresorhus/execa#reject

Reply:
cool let me change that.